### PR TITLE
Implement Line Item API

### PR DESCRIPTION
**Pull request by roycornelissen**
This PR implements the Line Item API. This API is a bit "strange" compared to others, since some request and response types deviate from regular list or plain hubspot entities, so I had to add some new logic to the serializer. I took the liberty to do some small refactorings as well.

Fixes/implements #47

### CODE REVIEW

File: src/Core/Requests/RequestSerializer.cs [hunk 1]
```diff
@@ -64,7 +65,8 @@ public virtual string SerializeEntity(object obj)
         /// <returns>The serialized list of entities</returns>
         public virtual string SerializeEntities<T>(List<T> objs)
         {
-            var result = "[";
+            var result = new StringBuilder("[");
```
Comment (nover) [inline src/Core/Requests/RequestSerializer.cs:68]: **praise**: nice optimization, thanks 👍

File: src/LineItem/Interfaces/LineItemRequestOptions.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        /// </summary>
+        /// <remarks>
+        /// The return DTO from List contains the current "offset" that you can inject into your next list call 
+        /// to continue the listing process
+        /// </remarks>
+        public long? Offset { get; set; } = null;
+        public List<string> PropertiesToInclude { get; set; } = new List<string>();
+    }
+
+    public class LineItemGetRequestOptions
```
Comment (nover) [inline src/LineItem/Interfaces/LineItemRequestOptions.cs:19]: **nitpick(non-blocking)** : I prefer one file one class but in this instance I can make an exception.

**Review by nover — APPROVED** [review]
Overall I think it looks good and can be merged 💯

File: src/LineItem/Interfaces/LineItemRequestOptions.cs (hunk 2 above)
Comment (roycornelissen) [reply src/LineItem/Interfaces/LineItemRequestOptions.cs:19]: Yeah I agree. Actually I wasn’t sure, but I noticed request options combined in one other place too so I assumed it as the standard.

Comment (nover) [reply src/LineItem/Interfaces/LineItemRequestOptions.cs:19]: perhaps I have missed that in a previous PR or just got lazy 👀  Candidate for refactor at a later point.

**Pull request merged**